lib/glific/providers/gupshup/partner/partner_api.ex

```diff
@@ -112,8 +112,8 @@ defmodule Glific.Providers.Gupshup.PartnerAPI do
   """
   @spec apply_for_template(non_neg_integer(), map) :: tuple()
   def apply_for_template(org_id, payload) do
+    payload = Map.put_new(payload, "allowTemplateCategoryChange", true)
```

Comment:
line 115 and 116 looks similar, wanna merge both?
    payload =
      payload
      |> Map.put("allowTemplateCategoryChange", true)
      |> Map.put(payload, "appId", app_id!(org_id))